### ISSUE TITLE
vertical distribution

### ISSUE THREAD

**Issue by Ronadlo7** [shuhart/StepView#43]
this is a cool library,
How to set the vertical distribution？
thank you

**Comment by shuhart** [comment]
This is not supported yet. Maybe in the near future if demand is enough.

**Comment by MSKM-Synergy** [comment]
yes i would also like it to have a vertical distribution and can you tell that can i add custom step numbers on the steps?

**Comment by pratikbutani** [comment]
Yes, I also want it vertical.

**Comment by niqiu1683** [comment]
Yes, I also want it vertical support. 
Are you developing this feature recently? if not yet, I hope to contribute some code if you permit. I am studying your code these days.

**Comment by shuhart** [comment]
Yes, that would be faster than waiting for me. as for your questions I have following suggestions based on official material guidelines https://material.io/archive/guidelines/components/steppers.html#steppers-types-of-steppers:

- Place a text on the right of a circle.
- To measure and draw the multiline text you have to use StaticLayout. You can find an example in my code.
-  Do not constraint the width of text. Just draw it in multilines considering a free space of the view itself.

**Comment by JibuVarghese** [comment]
Yes, I also vertical distribution.

**Comment by JibuVarghese** [comment]
Yes, I also want vertical distribution.
